apps/webcert/src/feature/certificate/UnifiedEdit/UeDateRange/UeDateRange.tsx

```diff
@@ -165,7 +165,9 @@ const UeDateRange: React.FC<Props> = ({ question, disabled }) => {
             textInputOnBlur={handleToTextInputOnBlur}
             textInputOnKeyDown={handleToTextInputOnKeyDown}
             textInputDataTestId={`tom${config.id}`}
-            displayValidationErrorOutline={getShouldDisplayValidationErrorOutline(config.id, 'tom')}
+            displayValidationErrorOutline={
+              getShouldDisplayValidationErrorOutline(config.id, 'tom') || getShouldDisplayValidationErrorOutline(config.id, 'to')
+            }
```

Comment:
I think there was a marking for the whole period that was missing too, besides "to".

Reply:
Nice catch! Updated and now the error is displayed correclty

![image](https://github.com/user-attachments/assets/be32cd37-c3db-4822-8353-40f70ce94571)